parsec/interfaces/dtd/insert_function.c

```diff
@@ -2447,16 +2447,16 @@ int parsec_dtd_task_class_add_chore(parsec_taskpool_t *tp,
     parsec_dtd_task_class_t *dtd_tc = (parsec_dtd_task_class_t*)tc;
 
     /* We assume that incarnations is big enough, because it has been pre-allocated
-     * with 7 chores, as this is a DTD task class */
+     * with PARSEC_DEV_MAX_NB_TYPE+1 chores, as this is a DTD task class */
     pincarnations = (__parsec_chore_t **)&dtd_tc->super.incarnations;
```

Comment:
@bosilca why is it `pincarnations = (__parsec_chore_t **)&dtd_tc->super.incarnations;` and not `pincarnations = (__parsec_chore_t **)dtd_tc->super.incarnations;` I don't see that `dtd_tc->super.incarnations` is assigned a new value and instead all accesses have to add another dereference `(*pincarnations)[i]`...

Reply:
Fixed.